addons/spark/0.x/spark-1.yaml

```diff
@@ -0,0 +1,36 @@
+---
+apiVersion: kubeaddons.mesosphere.io/v1beta1
+kind: Addon
+metadata:
+  name: spark
+  labels:
+    kubeaddons.mesosphere.io/name: spark
+    kubeaddons.mesosphere.io/hack-requires-defaultstorageclass: "true"
+  annotations:
+    stage.kubeaddons.mesosphere.io/spark: Beta
+    catalog.kubeaddons.mesosphere.io/addon-revision: "0.2.0-1"
+    appversion.kubeaddons.mesosphere.io/spark: "0.2.0"
+spec:
+  kubernetes:
+    minSupportedVersion: v1.15.0
+  cloudProvider:
+    - name: aws
+      enabled: true
+    - name: azure
+      enabled: true
+    - name: gcp
+      enabled: true
+    - name: docker
+      enabled: true
+    - name: none
+      enabled: true
+  requires:
+    - matchLabels:
+        # requires ServiceMonitor CRD to be present
+        kubeaddons.mesosphere.io/name: prometheus
+  kudoReference:
+    package: spark
+    repo: https://kudo-repository.storage.googleapis.com/0.10.0
```

Comment:
shouldn't the KUDO version be `0.10.1`?

Reply:
Yes please

Reply:
The `https://kudo-repository.storage.googleapis.com/0.10.0` is correct and will continue like that. KUDO versions >= `0.10.0` are pointing to `https://kudo-repository.storage.googleapis.com/0.10.0` 
we still need to discuss around when to move back `https://kudo-repository.storage.googleapis.com` and this is related around having a defined deprecation lifecycle